client/my-sites/checkout/checkout-thank-you/header.jsx

```diff
@@ -59,7 +57,6 @@ export class CheckoutThankYouHeader extends PureComponent {
 		translate: PropTypes.func.isRequired,
 		_n: PropTypes.func.isRequired,
 		upgradeIntent: PropTypes.string,
-		isRedesignV2: PropTypes.bool,
```

Comment:
We no longer need the `isRedesignV2` check for rendering `<CheckoutThankYouHeader>`. All redesigned congrats pages should not be using `<CheckoutThankYouHeader>` at all.